### PR TITLE
Starter Code Base + Custom Title Screen

### PR DESCRIPTION
**Pull request by MatthewHightech**
### The features added to it includes
- A homepage with the old open space logo. 
- A button that says "start game" 

### Nothing has been styled or formatted yet, and the button has no action, but it can be deployed to a local server for future testing and has no errors at the moment.

### CODE REVIEW

**Comment by FlyteWizard** [comment]
Hey! Would it be possible to include a `README.md` and `LICENSE` to this PR? 
The README.md would include instructions on getting the project running locally, and other important information. The LICENSE would probably be MIT, but you can discuss more about this before adding the document.

**Comment by MatthewHightech** [comment]
> Hey! Would it be possible to include a `README.md` and `LICENSE` to this PR?
> The README.md would include instructions on getting the project running locally, and other important information. The LICENSE would probably be MIT, but you can discuss more about this before adding the document.

Thanks for the reminder :) I just added a commit with an MIT licence, as well as a basic readme with instructions for local deployment.

File: src/app/home/home.page.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<div>
+    <ion-img src="..\..\assets\OldOpenSpaceLogo.png"></ion-img>
+
+<ion-button color="dark">Start Game</ion-button>
+
+<app-game-screen></app-game-screen>
```
Comment (FlyteWizard) [inline src/app/home/home.page.html:6]:
```suggestion
    <ion-button color="dark">Start Game</ion-button>

    <app-game-screen></app-game-screen>
```

File: src/app/home/home.page.scss [hunk 2]
```diff
@@ -0,0 +1,35 @@
+#container {
+  text-align: center;
+
```
Comment (FlyteWizard) [inline src/app/home/home.page.scss:3]:
```suggestion
```

File: src/app/home/home.page.scss [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+#container strong {
+  font-size: 20px;
+  line-height: 26px;
+}
+
+#container p {
+  font-size: 16px;
+  line-height: 22px;
+
```
Comment (FlyteWizard) [inline src/app/home/home.page.scss:19]:
```suggestion
```

File: src/app/home/home.page.scss [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  color: #8c8c8c;
+
```
Comment (FlyteWizard) [inline src/app/home/home.page.scss:21]:
```suggestion
```

**Review by FlyteWizard — APPROVED** [review]
This looks good, and I was able to get the app running 😄 

## Title Screen
Sample of Project

A few minor changes for syntax, but I don't see any problems with the code base. 👍

File: src/app/home/home.page.scss (hunk 4 above)
Comment (MatthewHightech) [reply src/app/home/home.page.scss:21]: Thanks for all the help @FlyteWizard :) We didn't have time to format the syntax today, but hopefully, future edits will be all cleaned up. I'm gonna merge for tomorrow. Thanks again!

**Pull request merged**